### ISSUE TITLE
Global styles is broken in firefox 36

### ISSUE THREAD

**Issue by helllamer** [JasonBarnabe/stylish#224]
As pointed in comments in addons.mozilla.org, there are a problem with global styles in Firefox 36+. https://addons.mozilla.org/ru/firefox/addon/stylish/reviews/

Steps to reproduce:
1. Install global dark style https://userstyles.org/styles/47391/midnight-surfing-alternative
2. Activate it.
3. Open or reload any page, for example https://github.com/JasonBarnabe/stylish
4. Blank page on the screen and several errors in firefox error console:
```
TypeError: stylishOverlay.currentURI is undefined
TypeError: stylishOverlay.STRINGS.getString is not a function
```
and possibly others.

**Comment by JasonBarnabe** [comment]
See discussion at https://forum.userstyles.org/discussion/44470/some-styles-suddenly-screwing-up-on-firefox-aurora

tl;dr: Firefox changed some internals, that style is opting in to "let's screw with the internals" mode, so things broke. The JS errors are unrelated.

**Comment by digitalcosmos** [comment]
A quick fix is to remove `/* AGENT_SHEET */` from the style.
At least until Firefox gets their sh*t together.
